Dockerfile

```diff
@@ -30,6 +30,17 @@ RUN  \
 	rm $GORELEASER_DOWNLOAD_FILE && \
 	goreleaser -v
 
+# install cosign
+ARG COSIGN_VERSION=1.2.1
+ARG COSIGN_SHA=490cb1941aa317cd24a0bd9f2fe38932805dbaaba0ae89c12ec8138d15bdd8a0
+RUN  \
+	COSIGN_DOWNLOAD_FILE=cosign-linux-amd64 && \
+	COSIGN_DOWNLOAD_URL=https://github.com/sigstore/cosign/releases/download/v${COSIGN_VERSION}/${COSIGN_SHA} && \
```

Comment:
I think there is no release file named like this, right?

Reply:
no no i messed up :(